### PR TITLE
Support converting KTypes and KotlinTypes in the same string represen…

### PR DESCRIPTION
**Pull request by nbirillo**
…tation

### CODE REVIEW

File: reflekt-plugin/src/main/kotlin/io/reflekt/plugin/generation/ir/util/IrBuilderUtil.kt [hunk 1]
```diff
@@ -45,6 +45,8 @@ fun IrBuilderWithScope.irKClass(symbol: IrClassSymbol) =
 fun IrBuilderWithScope.irKFunction(type: IrType, symbol: IrFunctionSymbol): IrFunctionReference {
     require(type is IrSimpleType)
     val kFunctionType = IrSimpleTypeImpl(
+        // TODO: replace into this one in the next Kotlin version since API will be changed
```
Comment (elena-lyulina) [inline reflekt-plugin/src/main/kotlin/io/reflekt/plugin/generation/ir/util/IrBuilderUtil.kt:48]: into -> to

File: reflekt-plugin/src/main/kotlin/io/reflekt/plugin/utils/Util.kt [hunk 2]
```diff
@@ -80,3 +83,19 @@ fun <T : Enum<T>> enumToRegexOptions(values: Array<T>, transform: T.() -> String
 
 fun <T : Enum<T>> String.toEnum(values: Array<T>, transform: T.() -> String): T =
     values.first { it.transform() == this }
+
+fun KotlinType.stringRepresentation(): String {
```
Comment (elena-lyulina) [inline reflekt-plugin/src/main/kotlin/io/reflekt/plugin/utils/Util.kt:87]: why it's not in TypeStringRepresentation file?

File: reflekt-plugin/src/test/resources/io/reflekt/plugin/util/type/representation/kotlinTypes/Util.kt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package io.reflekt.plugin.util.type.representation.kotlinTypes
+
+
+
+/**
+ * Function to test Ia string representation of KotlinTypes, a test key is passed as argument
```
Comment (elena-lyulina) [inline reflekt-plugin/src/test/resources/io/reflekt/plugin/util/type/representation/kotlinTypes/Util.kt:6]: Ia?

**Review by elena-lyulina — CHANGES_REQUESTED** [review]
Cool, thanks!

File: reflekt-plugin/src/main/kotlin/io/reflekt/plugin/utils/Util.kt (hunk 2 above)
Comment (nbirillo) [reply reflekt-plugin/src/main/kotlin/io/reflekt/plugin/utils/Util.kt:87]: Since we don't have access to `KotlinType` in the TypeStringRepresentation file

**Pull request merged**